src/main/java/org/vandeseer/easytable/TableDrawer.java

```diff
@@ -130,7 +134,7 @@ public void draw(Supplier<PDDocument> documentSupplier, Supplier<PDPage> pageSup
                 pageToDrawOn = document.getPage(document.getNumberOfPages() - 1);
             }
 
-            try (final PDPageContentStream newPageContentStream = new PDPageContentStream(document, pageToDrawOn, APPEND, false)) {
+            try (final PDPageContentStream newPageContentStream = new PDPageContentStream(document, pageToDrawOn, APPEND, compress)) {
```

Comment:
Theoretically one could also create a method for creating the content stream – which could be subclassed and completely adjusted to one's needs.  But I think for now just setting the flag should be sufficient. 

Also I don't like the fact that we rely on `boolean` defaulting to `false`.  But again I will leave it for now and refactor it later should there be more requirements in this regard.